src/share/poudriere/include/fs.sh

```diff
@@ -34,10 +34,7 @@ createfs() {
 
 	if [ -n "${fs}" -a "${fs}" != "none" ]; then
 		msg_n "Creating ${name} fs at ${mnt}..."
-		if ! zfs create -p \
-			-o compression=lz4 \
-			-o atime=off \
-			-o mountpoint=${mnt} ${fs}; then
+		if ! zfs create -p ${fs}; then
```

Comment:
Yeah let's drop the `-p` here. If it fails trying to create outside of the ZROOTFS then something else is broken.